src/lib.rs

```diff
@@ -1,11 +1,12 @@
+#![cfg(any(target_os = "macos", target_os = "ios"))]
+#![allow(clippy::missing_safety_doc, clippy::let_unit_value)]
+
 #[macro_use]
 extern crate objc;
 
 use objc::runtime::Object;
 
-#[cfg(target_os = "macos")]
 pub mod appkit;
-#[cfg(target_os = "ios")]
 pub mod uikit;
```

Comment:
@msiglreith I thought I left a comment here but that [was on your `0.4` PR](https://github.com/norse-rs/raw-window-metal/pull/4#discussion_r932630819), thoughts?

> @msiglreith I've been looking into reviving the related `ash-window` PR now that `raw-window-handle 0.5` is out _and_ used in `winit 0.27`, and stumbled upon these cfgs. According to https://github.com/rust-windowing/raw-window-handle/pull/70/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759 (surprisingly the author detailed/justified nothing of this in PR/commit messages...) it should be possible to use these across `macos` _and_ `ios` now when targeting "Mac Catalyst"?

Reply:
As it passes builds I'm happy with it, given these are not strictly iOS/macOS anymore (as it was in the past).

Reply:
Note that I moved these to the top of the file with `#![cfg(any(target_os = "macos", target_os = "ios"))]`, since it's still all limited to Apple platforms.